### PR TITLE
fix(java-plugin): Disable java semanticHighlighting by default

### PR DESCRIPTION
**Pull request by sunix**
### What does this PR do?
Quick fix for syntax highlighting. Semantic highlighting is not properly enriching the default syntax highlighting. Disabling it for now.

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18448

### How to test this PR?

TODO

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: sidecars/java/etc/entrypoint.sh [hunk 1]
```diff
@@ -27,4 +27,9 @@ if [ "${USER_ID}" -ne 0 ] && command -v sudo >/dev/null 2>&1 && sudo -n true > /
     sudo chown "${USER_ID}:${GROUP_ID}" /projects
 fi
 
+echo 'Setting "java.semanticHighlighting.enabled" to false'
```
Comment (benoitf) [inline sidecars/java/etc/entrypoint.sh:30]: how does it work with multi-root workspaces ? 
I think these kind of settings may be ignored
cc @RomanNikitenko

Comment (sunix) [reply sidecars/java/etc/entrypoint.sh:30]: How will multi-root workspaces deal with "workspace" settings ? like the one defined in the devfile ?

Comment (tsmaeder) [reply sidecars/java/etc/entrypoint.sh:30]: https://github.com/eclipse/che/issues/17975#issuecomment-760229209 and the follow-up comment. Answer: we need some new code.

Comment (sunix) [reply sidecars/java/etc/entrypoint.sh:30]: But same is applying for the settings you apply in the devfile like: https://github.com/eclipse/che-devfile-registry/blob/dcfdacf3954e30a25a8af3c7bd497b193d24f14a/devfiles/quarkus/devfile.yaml#L59-L62
So the strategy needs to be covered in the multi-root PR

**Comment by sunix** [comment]
What I can also do is disable `editor.semanticHighlighting.enabled` in che-theia there: https://github.com/eclipse/che-theia/blob/master/generator/src/templates/assembly-package.mst#L11-L15

File: sidecars/java/etc/entrypoint.sh (hunk 1 above)
Comment (RomanNikitenko) [reply sidecars/java/etc/entrypoint.sh:30]: I posted my thoughts about it here https://github.com/eclipse/che/issues/17212#issuecomment-778575536
Please share your opinion

**Comment by sunix** [comment]
won't merge in favor of https://github.com/eclipse/che-theia/pull/993

**Pull request closed without merging**